.changeset/many-forks-exercise.md

```diff
@@ -0,0 +1,6 @@
+---
+"@wso2is/console": patch
+"@wso2is/i18n": patch
```

Comment:
bump myaccount and identity-apps-core as well

Reply:
Done [49e6a46](https://github.com/wso2/identity-apps/pull/4669/commits/49e6a466b17f41cb3bd9d99a1997c40bb0936c1b)